SourceKitStressTester/Sources/StressTester/ActionGenerators.swift

```diff
@@ -503,6 +503,32 @@ private struct ActionToken {
         return nil
       }
     }
+
+    // Avoid matching for underscored attributes since we hide completions for
+    // those.
+    do {
+      var parent = parent
+      if let label = parent.as(LabeledExprSyntax.self),
+         let labelParent = label.parent, label.label == token {
+        parent = labelParent
+      } else if let nextParent = parent.parent {
+        parent = nextParent
+        if let label = parent.as(LabeledExprSyntax.self),
+           let labelParent = label.parent {
+          parent = labelParent
+        }
+      }
+      if let exprList = parent.as(LabeledExprListSyntax.self),
+         let listParent = exprList.parent {
+        parent = listParent
+      }
+      if let attr = parent.as(AttributeSyntax.self),
+         let ident = attr.attributeName.as(IdentifierTypeSyntax.self),
+         ident.name.text.hasPrefix("_") {
+        return nil
+      }
```

Comment:
Would it make sense to just generalize this and not run actions inside underscored attributes using something like

```swift
if parent.ancestorOrSelf(mapping: { node in
  node.as(AttributeSyntax.self).attributeName.as(IdentifierTypeSyntax.self).hasPrefix("_")
}) != nil {
  return nil
}
```